src/main/resources/static/api.yaml

```diff
@@ -723,4 +765,64 @@ components:
         telephone:
           type: string
         type:
-          type: string
\ No newline at end of file
+          type: string
+    ConsentRequest:
+      type: object
+      properties:
+        consent:
+          type: object
+          properties:
+            purpose:
+              type: object
+              properties:
+                text:
+                  type: string
+                code:
+                  type: string
+                refUri:
+                  type: string
+                  ormat: uri
+            patient:
+              allOf:
+                - $ref: '#/components/schemas/ConsentManagerPatientID'
+            hip:
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/OrganizationRepresentation'
+                - xml:
+                    name: hip
+            hiu:
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/OrganizationRepresentation'
+                - xml:
+                    name: hiu
+            requester:
+              $ref: '#/components/schemas/Requester'
+            hitypes:
+              type: array
+              items:
+                $ref: '#/components/schemas/HITypeEnum'
+            permission:
```

Comment:
frequency should be part of the permission. 
      "frequency": {
          "unit": "DAY" , //from a defined list
          "value": 0
        }